lib/assets/javascripts/twine.js.coffee

```diff
@@ -315,7 +315,7 @@ setupAttributeBinding('innerHTML', 'unsafe-html')
 
 preventDefaultForEvent = (event) ->
   (event.type == 'submit' || event.currentTarget.nodeName.toLowerCase() == 'a') &&
-  Twine.getAttribute(event.currentTarget, 'allow-default') != '1'
+  Twine.getAttribute(event.currentTarget, 'allow-default') in ['false', false, 0, undefined, null]
```

Comment:
We would still have to check for falsey values right? So that would end up looking like 

```
(event.type == 'submit' || event.currentTarget.nodeName.toLowerCase() == 'a') &&
Twine.hasAttribute(event.currentTarget, 'allow-default') &&
Twine.getAttribute(event.currentTarget, 'allow-default') in ['false', false, 0, undefined, null]
```

right?